pkg/cluster/install.go

```diff
@@ -373,6 +375,12 @@ func (m *manager) startInstallation(ctx context.Context) error {
 			doc.OpenShiftCluster.Properties.Install = &api.Install{
 				Now: time.Now().UTC(),
 			}
+
+			// TODO remove this when introducing the BYONSG CLI option
+			if feature.IsRegisteredForFeature(m.subscriptionDoc.Subscription.Properties, api.FeatureFlagBYONsg) {
+				m.log.Logger.Info("BYO NSG feature flag is on")
+				doc.OpenShiftCluster.Properties.NetworkProfile.PreconfiguredNSG = api.PreconfiguredNSGEnabled
+			}
```

Comment:
move this into pkg/frontend/openshiftcluster_putorpatch.go` around the `determineOutboundType` function.